src/overview/selectors.js

```diff
@@ -175,3 +166,21 @@ export const isEmptyQuery = createSelector(
         !showOnlyBookmarks &&
         !tags.length,
 )
+
+export const filterPopup = state => overview(state).filterPopup
+export const filterTags = state => overview(state).filterTags
+export const filterDomains = state => overview(state).filterDomains
+
+/**
+ * Selector to toggle clear filter button
+ * As new filters are added, corersponding changes need to made to this function
+ */
+export const isClearFilterButtonShown = createSelector(
+    showOnlyBookmarks,
+    filterTags,
+    filterDomains,
+    (showOnlyBookmarks, filterTags, filterDomains) =>
+        !!showOnlyBookmarks ||
+        Boolean(filterTags.length) ||
```

Comment:
Can just use double-bang `!!` here to coerce into a `boolean` type. Should be same result, but to be consistent with rest of code